cmd/draupnir-finalise-image

```diff
@@ -94,8 +94,26 @@ wal_level = 'minimal'
 fsync = 'off'
 EOF
 
+LOG_FILE="/var/log/postgresql/image_${ID}"
+
 # Start postgres
-sudo -u postgres $PG_CTL -w -D "$UPLOAD_PATH" -o "-p $PORT" -l "/var/log/postgresql/image_$ID" start
+sudo -u postgres $PG_CTL -w -D "$UPLOAD_PATH" -o "-p $PORT" -l "${LOG_FILE}" start
+
+# We need to wait for postgres to boot and announce that the recovery has
+# completed. Ideally WAL recovery shouldn't take long, but for high volume
+# databases Postgres needs a window to catch-up from the last checkpoint.
+TIMEOUT=300 # 5m
```

Comment:
Is 5m a safe enough value, given that it took 3m to begin accepting queries this morning?

Reply:
Probably not then. 10m sounds about right?